main.cpp

```diff
@@ -83,44 +83,95 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 
 /*
  1)
- */
-
+ */
+
+ bool runMarathon(int speed = 10, int lenght = 20)
+{
+    ignoreUnused(speed, lenght); return 
+    { 
+        };
```

Comment:
Fix this formatting.
Look at the example if you're not sure what your code should look like.
- [ ] revise